proto/launch/tx.proto

```diff
@@ -36,6 +36,7 @@ message MsgRequestAddValidator {
 
 message MsgRequestAddValidatorResponse {
```

Comment:
I think we can create a single proto message `MsgRequestResponse` for all request msgs